srtcore/packet.cpp

```diff
@@ -210,6 +210,7 @@ void CPacket::allocate(size_t alloc_buffer_size)
         delete [] m_pcData;
     }
     m_PacketVector[PV_DATA].set(new char[alloc_buffer_size], alloc_buffer_size);
+    setLength(alloc_buffer_size);
     m_data_owned = true;
```

Comment:
`setLength(alloc_buffer_size);` should be removed

Reply:
welcome to automatic merges...